src/tokenizer/stopwords/english.txt

```diff
@@ -0,0 +1,312 @@
+
+ | An English stop word list. Comments begin with vertical bar. Each stop
+ | word is at the start of a line.
+
+ | Many of the forms below are quite rare (e.g. "yourselves") but included for
+ |  completeness.
+
+           | PRONOUNS FORMS
+             | 1st person sing
+
+i              | subject, always in upper case of course
+
+me             | object
+my             | possessive adjective
+               | the possessive pronoun `mine' is best suppressed, because of the
+               | sense of coal-mine etc.
+myself         | reflexive
+             | 1st person plural
+we             | subject
+
+| us           | object
+               | care is required here because US = United States. It is usually
+               | safe to remove it if it is in lower case.
+our            | possessive adjective
+ours           | possessive pronoun
+ourselves      | reflexive
+             | second person (archaic `thou' forms not included)
+you            | subject and object
+your           | possessive adjective
+yours          | possessive pronoun
+yourself       | reflexive (singular)
+yourselves     | reflexive (plural)
+             | third person singular
+he             | subject
+him            | object
+his            | possessive adjective and pronoun
+himself        | reflexive
+
+she            | subject
+her            | object and possessive adjective
+hers           | possessive pronoun
+herself        | reflexive
+
+it             | subject and object
+its            | possessive adjective
+itself         | reflexive
+             | third person plural
+they           | subject
+them           | object
+their          | possessive adjective
+theirs         | possessive pronoun
+themselves     | reflexive
+             | other forms (demonstratives, interrogatives)
+what
+which
+who
+whom
+this
+that
+these
+those
```

Comment:
I'm ok with this list, although it is still a tad long to my liking I think.

For reference, the one in Lucene 
https://cs.github.com/apache/lucene/blob/682e5c94e83320fee929a16f375fd0ebe4ddc99c/lucene/analysis/common/src/java/org/apache/lucene/analysis/en/EnglishAnalyzer.java?q=EnglishAnalyzer+org%3Aapache+#L46-L51

Reply:
Lucene does not seem to use Snowball for English, only for other languages like German.

I can drop the English one and replace it by the open-coded list from Lucene? (Apache Licensed should be compatible I think?)

Reply:
ok for me.

Reply:
I noticed that the current stop word list for English already is the one from Lucene and hence made the change purely additive. (I did opt to make the `StopWordFilter::english` method public though for consistency. I am also not really convinced the `Default` impl but did not opt to change it.)